docs/src/whatsnew/3.3.rst

```diff
@@ -52,6 +52,14 @@ v3.3.1 |build_date| [unreleased]
     #. `@pp-mo`_ ensured that :data:`iris.cube.Cube.cell_methods` must always be an iterable
        of :class:`iris.coords.CellMethod` objects (:pull:`4933`).
 
+    #. `@trexfeathers`_ advanced the Cartopy pin to ``>=0.21``, as Cartopy's
+       change to default Transverse Mercator projection affects an Iris test.
+       See `SciTools/cartopy@fcb784d`_ and `SciTools/cartopy@8860a81`_ for more
+       details. (:pull:`4968`)
```

Comment:
I note you've referenced the PR changes that were applied to main as opposed to using this PR's number. I suppose that makes sense as that was where the conversation happened but I think it may get confusing if we were to try to track what happened from the whats new?

Reply:
How silly of me 😳